.env.example

```diff
@@ -41,6 +41,7 @@ OPENSEARCH_URL=http://opensearch:9200
 
 CACHE_HOST=redis
 CACHE_SCHEME=redis
+# WP_REDIS_USE_RELAY=true
```

Comment:
Don't we need this to be true?

Reply:
I'm thinking that Relay may be causing CPU spikes. With that in mind, it's optional, with default being unset.

I'm not certain, if you would prefer , I can change it.